### DISCUSSION TITLE
fixed! SMD72 MODBUS compile parameters

### DISCUSSION THREAD

**Discussion by acht11** [arendst/Tasmota#17597] [Q&A] [answered]
![Tasmota](https://user-images.githubusercontent.com/86398696/210578098-83f6f89a-7004-4708-bad2-665ba6733c37.png)
Hello guys, I am searching for the compiler parameter, to be able to use more than 20 Decoders. In the documentation, see above, the parameter was forgot. Anybody knows him?
Thanks in advance

**Comment by gemu2015** [comment, marked as answer]
In latest dev you may define as much decode lines as needed with dynamic allocation
No more need to define maxvars

**Comment by acht11** [comment]
Thanks a lot. Thats it.
Greetings
Jo

**Reply by barbudor** [reply]
@acht11 May be you can suggest a doc change by clicking on the pencil button on the top right of the page to remove that no more necessary comment
